src/main/resources/templates/pps/home.html

```diff
@@ -14,105 +14,97 @@
-                            <p class="govuk-body">Account name: Late Filing Penalties Receipts</p>
-                        </div>
-                        <p class="govuk-body"></p>
-                        <p class="govuk-body">Email the LFP finance team at Companies House to tell them you've paid.
-                            Include in your email the:</p>
-                        <ul class="govuk-list govuk-list--bullet" id="list-requirements-overseas-payments">
-                            <li>company number</li>
-                            <li>company name</li>
-                            <li>reference number on the penalty notice</li>
-                            <li>amount paid</li>
-                            <li>date paid</li>
-                        </ul>
-                        <div class="govuk-details__text" aria-hidden="true" id="companies-house-finance-team-email">
-                            <p class="govuk-body">LFP finance team, Companies House <a
-                                    href="mailto:lfpfinance@companieshouse.gov.uk">lfpfinance@companieshouse.gov.uk</a>
-                            </p>
-                        </div>
-                        <p class="govuk-body"></p>
-                        <h2 class="govuk-heading-m">Pay by cheque</h2>
-                        <p class="govuk-body">You can pay by cheque through the post. Make the cheque payable to
-                            'Companies House only' and send it along with the slip on the penalty notice ('remittance
-                            advice') to Companies House.</p>
-                        <div class="govuk-details__text" aria-hidden="true" id="companies-house-address-cheque">
-                            <p>Companies House
-                                <br>
-                                PO Box 710
-                                <br>
-                                Crown Way
-                                <br>
-                                CF14 3UZ
-                            </p>
+                <div class="govuk-!-margin-top-6">
+                    <details class="govuk-details">
+                        <summary class="govuk-details__summary">
+                            <span class="govuk-details__summary-text"> I want to appeal a penalty </span>
+                        </summary>
+                        <div class="govuk-details__text">
+                            <h3 class="govuk-heading-s">If you have a late filing penalty</h3>
+                            This option is only for penalties for late filing of accounts. You can <a
+                                href="https://www.gov.uk/appeal-a-penalty-for-filing-your-company-accounts-late"
+                                class="govuk-link">appeal a late filing penalty (for filing accounts late) through
+                            Companies House</a>. You must show that the reasons are exceptional.<p></p>
```

Comment:
Please can you check your paragraph tags throughout where we have `<p></p>`.
Should we be putting the paragraph text inside them?